### PR TITLE
Include SPScript

### PR DESCRIPTION
**Pull request by DroopyTersen**
Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/DroopyTersen/spscript
 * Official website (optional, not the repository): http://droopytersen.github.io/spscript/
 * NPM package url (optional): https://www.npmjs.com/package/spscript
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 7
   - Count of stars: 8
   - Count of forks: 5
 * NPM download stats (optional):
   - Downloads in the last week: 170
   - Downloads in the last month: 300
   - Dowloads last year: 6000

# Essential checklist
 * [x ] I'm the author of this library
   * [ x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x ] This lib was not found on cdnjs repo
 * [x ] No already exist / duplicated issue and PR
 * [ x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/spscript/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "keywords": [
+    "sharepoint",
+    "rest",
+    "droopy",
+    "office",
+    "office-365"
+    "office365"
+  ],
+  "author": "Andrew Petersen",
+  "bugs": {
```
Comment (pvnr0082t) [inline ajax/libs/spscript/package.json]: @DroopyTersen Please remove `"bugs"` field.

File: ajax/libs/spscript/package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "url": "https://github.com/DroopyTersen/spscript/issues"
+  },
+  "homepage": "https://github.com/DroopyTersen/spscript"
```
Comment (pvnr0082t) [inline ajax/libs/spscript/package.json]: We'll use git repo as homepage by default if the custom homepage is not found, so there is no need to set homepage to its git repo. Or you can use https://droopytersen.github.io/spscript/ as `homepage`.

File: ajax/libs/spscript/package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "spscript",
+  "version": "2.0.0",
+  "description": "SharePoint 2013 Rest Api Wrappers",
+  "filename": "dist/v2/spscript.min.js",
```
Comment (pvnr0082t) [inline ajax/libs/spscript/package.json]: We can just use `"spscript.min.js"`.

File: ajax/libs/spscript/package.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "spscript",
+  "version": "2.0.0",
```
Comment (pvnr0082t) [inline ajax/libs/spscript/package.json]: Please remove version field if you want to add library via single package.json.

File: ajax/libs/spscript/package.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    "office365"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/droopytersen/spscript.git",
+    "fileMap": [{
+        "basePath": "dist/v2",
+        "files": [
+          "spcript.js",
+          "spscript.min.js"
```
Comment (pvnr0082t) [inline ajax/libs/spscript/package.json]: Let's use `"**/spscript*"` instead of these two rule.

File: ajax/libs/spscript/package.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  "keywords": [
+    "sharepoint",
+    "rest",
+    "droopy",
+    "office",
+    "office-365"
+    "office365"
+  ],
+  "autoupdate": {
```
Comment (pvnr0082t) [inline ajax/libs/spscript/package.json]: There is no git tags on GitHub repo, so git auto-update config will not work. Let's use npm auto-update config instead.

**Comment by PeterDaveHello** [comment]
Hello @DroopyTersen, are you still there and want to finish this PR? Thanks.

**Comment by PeterDaveHello** [comment]
@DroopyTersen we're happy to help you add it if you want to host it on cdnjs.

**Pull request closed without merging**